fast_llm/data/dataset/gpt/config.py

```diff
@@ -160,6 +165,41 @@ class GPTBlendedDatasetConfig(BlendedDatasetConfig, GPTSampledDatasetConfig):
     datasets: list[GPTSampledDatasetConfig] = FieldUpdate()
 
 
+@config_class()
+class GPTComposedDatasetConfig(GPTIndexedDatasetConfig):
+    _abstract: typing.ClassVar[bool] = False
+    type_: typing.ClassVar[str | None] = "composed"
```

Comment:
```suggestion
class GPTConcatenatedMemmapConfig(GPTIndexedDatasetConfig):
    _abstract: typing.ClassVar[bool] = False
    type_: typing.ClassVar[str | None] = "concatenated_memmap"
```
for your convenience, so that we can merge this easily.